ruby/example_code/lambda/scenario_getting_started_functions.rb

```diff
@@ -0,0 +1,120 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# frozen_string_literal: true
+
+# To run this demo, you will need Ruby 2.6 or later, plus dependencies.
+# For more info, see:
+# https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/ruby/README.md
+
+require "aws-sdk-lambda"
+require "aws-sdk-cloudwatchlogs"
+require "aws-sdk-iam"
+require "logger"
+require "json"
+require "zip"
+require "cli/ui"
+require_relative("../../helpers/disclaimers")
+require_relative("../../helpers/decorators")
+require_relative("lambda_basics")
+require_relative("lambda_function")
+
+@logger = Logger.new($stdout)
+@logger.level = Logger::WARN
+
+# snippet-start:[ruby.example_code.lambda.Scenario_GettingStartedFunctions]
+
+# Runs the scenario.
+# @return [nil]
+def run_scenario
+  banner
+  puts "######################################################################################################".yellow
+  puts "#                                                                                                    #".yellow
+  puts "#                                          EXAMPLE CODE DEMO:                                        #".yellow
+  puts "#                                             AWS Lambda                                             #".yellow
+  puts "#                                                                                                    #".yellow
+  puts "######################################################################################################".yellow
+  puts ""
+  puts "You have launched a demo of AWS Lambda using the AWS for Ruby v3 SDK. Over the next 60 seconds, it will:"
```

Comment:
puts "You have launched a demo of AWS Lambda using the AWS SDK for Ruby (v3). Over the next 60 seconds, it will do the following:"